echopype/convert/set_groups_ek60.py

```diff
@@ -321,17 +321,18 @@ def set_platform(self, NMEA_only=False) -> xr.Dataset:
                             self.parser_obj.ping_time[ch],
                             {
                                 "axis": "T",
-                                "long_name": "Timestamps for position datagrams",
+                                "long_name": "Timestamps for platform motion and orientation data",
                                 "standard_name": "time",
-                                "comment": "Time coordinate corresponding to platform sensors.",
+                                "comment": "Time coordinate corresponding to platform motion and "
+                                "orientation sensors.",
                             },
                         ),
                         "time3": (
                             ["time3"],
                             self.parser_obj.ping_time[ch],
                             {
                                 "axis": "T",
-                                "long_name": "Timestamps for position datagrams",
+                                "long_name": "Timestamps for environmental data",
                                 "standard_name": "time",
                                 "comment": "Time coordinate corresponding to "
                                 "environmental variables.",
```

Comment:
I suggest the following to explicitly state the source of data and not to use "environmental data" because that is easily confused with the actual environmental data (which should be in the `Environment` group).
```suggestion
                                "long_name": "Timestamps for platform-related sampling environment",
                                "standard_name": "time",
                                "comment": "Time coordinate corresponding to platform-related sampling"
                                "environment. For EK60, this data is saved in the RAW0 sample datagrams"
                                "along with the backscatter data",
```